src/constants/constants.ts

```diff
@@ -126,7 +126,7 @@ export const outputContentTypeShowWarning = "showWarning";
 export const outputServiceLocalhost = "http://localhost:";
 export const msgContentProviderSqlOutputHtml = "dist/html/sqlOutput.ejs";
 export const contentProviderMinFile = "dist/js/app.min.js";
-export const untitledSaveTimeThreshold = 10.0;
+export const untitledSaveTimeThreshold = 50.0;
```

Comment:
In your testing, how high did the gap between events tend to be?  10ms to 50ms is a big jump.

Reply:
Just ran it 5 times in a row, here are the results:
Mac:
12 ms
14 ms
13 ms
8 ms
11 ms


Windows:
26 ms
29 ms
30 ms
41 ms
32 ms